frontend/pages/admin/IntegrationsPage/cards/MdmSettings/AppleMdmPage/components/modals/RenewCertModal/RenewCertModal.tsx

```
@@ -60,11 +60,9 @@ const RenewCertModal = ({
   }, [certFile, renderFlash, onCancel, onRenew]);
 
   const onDownloadError = useCallback(
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     (e: unknown) => {
       const msg = getErrorReason(e);
-
-      if (msg.toLowerCase().includes("email address is not valid")) {
+      if ((e as any)?.response?.headers?.["invalidEmailDomain"]) {
```

ESLint

[error] ["invalidEmailDomain"] is better written in dot notation.

(dot-notation)